src/mvt_sql.py

```diff
@@ -231,9 +205,9 @@ def get_vectors(minzoom, maxzoom, x, y, style, vec, extent):
             minzoom,
             x,
             y,
-            pixel_size_at_zoom(maxzoom, pxtolerance) ** 2,
+            pixel_size_at_zoom(maxzoom, 1) ** 2,
```

Comment:
while I was trying to figure out why 4th zoom tile size is near 1.5mb I discovered that most of the features are natural=water polygons which looks like points or completely not visible on 4th zoom so I think the threshold for area should be higher.

Reply:
I believe that useful way will be:
 - generate polygons via fast way
 - check that it is < 500 kb
 - if it is larger, re-run with buffer thing